wazuh/wazuh-kibana-app
# Fix references to Elasticsearch in plugin for Wazuh stack

### Description

Fixed references to Elasticsearch in Wazuh-stack plugin. Removed also the checkPattern function and its tests since it was not being used.

### Issues Resolved
[#4361]

### Evidence

Before

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/99441266/202715537-8b0e1113-ad04-4009-9a84-536443cfdbb8.png">

After

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/99441266/202715800-aa20c999-bbf6-4859-b7c4-315b9db9acbe.png">

### Steps to test

Go to the devTools and run in the console:

```
GET _cat/templates
DELETE _template/wazuh*
```

Then go to the health Check and check that it returns the following error:

`[Alerts index pattern] 4002 - Could not retrieve templates from Wazuh indexer due to An unknown error occurred when fetching templates from Wazuh indexer`

### Check List
